src/applications/survivors-benefits/config/form.js

```diff
@@ -80,6 +96,71 @@ const formConfig = {
   errorText: ErrorText,
   showReviewErrors: !environment.isProduction() && !environment.isStaging(),
   chapters: {
+    // Chapter 1 - Veteran Information
+    veteranInformation: {
```

Comment:
Can you restructure this section so it matches chapter for or military history? As the form grows, it is much easier to be able to just look through this file and see most of the form structure. I think would just need to break veteranInfo into two files.